src/settings/toml/manifest.rs

```diff
@@ -53,6 +54,10 @@ impl Manifest {
         Ok(manifest)
     }
 
+    pub fn new_from_string(serialized_toml: String) -> Result<Self, toml::de::Error> {
```

Comment:
a new constructor to allow in memory testing, otherwise our sample toml directory would get just massive.